admin/provisioner/provisioner.go

```diff
@@ -40,10 +41,13 @@ type Provisioner interface {
 
 type staticSpec struct {
 	Runtimes []*staticRuntime `json:"runtimes"`
+	// Map of runtimes by Region
+	runtimesByRegion map[string][]*staticRuntime
```

Comment:
Remove this and just iterate over in `Provision` (it's called rarely and the spec is small, so will be really fast)